src/components/Button/PlayButton.js

```diff
@@ -1,27 +1,25 @@
 import React, { PropTypes } from 'react';
+import styled, { css } from 'styled-components'
 
-/**
+const commonPlayBtnClasses = 'fa fa-play w3 h3 f3 absolute z-1 gray items-center justify-center br-pill pointer'
+const hoverPlayBtnClasses = `${commonPlayBtnClasses} bg-white-70`
+const playBtnClasses = `${commonPlayBtnClasses} hover-turquoise bg-white`
 
-.card-play-btn {
+const StyledPlayButton = styled.div`
   box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.17);
   left: 50%;
   margin-left: -34px;
   display: flex;
   transition: 150ms;
-}
-.card-play-btn:before {
-  position: relative;
-  left: 3px;
-}
-
- * */
-
-const commonPlayBtnClasses = 'fa fa-play w3 h3 f3 absolute z-1 gray items-center justify-center br-pill pointer card-play-btn'
-const hoverPlayBtnClasses = `${commonPlayBtnClasses} bg-white-70 o-0`
-const playBtnClasses = `${commonPlayBtnClasses} hover-turquoise bg-white`
+  &:before {
+    position: relative;
+    left: 3px;
+  }
+  ${props => props.hover ? 'opacity: 0;' : ''},
```

Comment:
Here, `props` is whatever passed into from the component that ultimately wraps the styled component we are currently making. We then apply `opacity: 0;` if `hover` is true.